### DISCUSSION TITLE
Astra with Curved Detector

### DISCUSSION THREAD

**Discussion by abbasmzs** [astra-toolbox/astra-toolbox#475] [Q&A]
I'm using astra, and I'm also using FDK algorithm with cone_vec projection geometry for reconstruction right now, I have good results but now I have some doubts about the functions that I have using, my data was obtained from a series "27(16*32)" of detectors that are in a curved shape and I have a cone beam ray (multi-slice fan beam);
Now my question and request is, what is the best way to reconstruct the images?

I want to know which "projection geometry" and which "algorithm" are the best choices for me with my conditions.
Also, I want to know if the conditions for curved detectors have been implemented in this astra-toolbox?

**Comment by askorikov** [comment]
Hi! Support for curved detectors is currently a work in progress, see also #139 and #162. If I understood you correctly, you have a fan beam after all, in which case I would advise using `fanflat` geometry and applying `FBP_CUDA` algorithm (or any of the iterative algorithms if you want to go that route) to each slice independently.
